pkgs/top-level/all-packages.nix

```diff
@@ -6151,6 +6151,8 @@ with pkgs;
 
   smokeqt = callPackage ../development/libraries/smokeqt { };
 
+  sn-cli = callPackage ../applications/misc/sn-cli { };
+
```

Comment:
I believe the merge conflict is in this file. New packages are generally put in pkgs/by-name.